### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by medzernik** [WootingKb/wooting-macros#211] [Announcements]
Please see https://github.com/WootingKb/wooting-macros/releases/tag/v1.1.0 for full changelog

# 1.1 Release
- [Added] UI Refresh - smoother animations, nicer colors.
- [Added] Macro search - search macros in all your collections.
- [Security] Updated libraries.
- [Fixed] Macros now work in games (be careful please!).
- [Fixed] Keycombos like CTRL+SHIFT+{Key} have been fixed.
- [Fixed] Many more bugfixes and improvements.
## More detailed changelog

- New UI features
- When recording a sequence, pressing LMB stops the recording
- Macro search
- Trigger key bind layout
- Security and library updates
- Better handling of errors
- New error and warning notifications in the UI
- New reset to default buttons in elements
- New collections search feature
- Per macro settings screen (features coming starting 1.2)
- Macros now work in games! Hooray (don't use it in multiplayer games pls thx)
- Delays have been lowered somewhat
- Macros with key combos (e.g. CTRL+SHIFT+KEY) should now work
![image](https://github.com/WootingKb/wooting-macros/assets/1900179/dc37b005-c623-4411-a33f-437d83eb53be)

## Generated changelog
* fixed categories showing when they shouldnt be by @khanguyen7 in https://github.com/WootingKb/wooting-macros/pull/140
* Bump openssl from 0.10.45 to 0.10.48 in /src-tauri by @dependabot in https://github.com/WootingKb/wooting-macros/pull/142
* Bump openssl from 0.10.45 to 0.10.48 in /wooting-macro-backend by @dependabot in https://github.com/WootingKb/wooting-macros/pull/141
* Library update by @medzernik in https://github.com/WootingKb/wooting-macros/pull/143
* Library update November by @medzernik in https://github.com/WootingKb/wooting-macros/pull/172
* 1.1 release by @medzernik in https://github.com/WootingKb/wooting-macros/pull/194
* Version bump to 1.1.0 by @medzernik in https://github.com/WootingKb/wooting-macros/pull/208
* Bump ip from 2.0.0 to 2.0.1 by @dependabot in https://github.com/WootingKb/wooting-macros/pull/204
* Changed log to new branch by @medzernik in https://github.com/WootingKb/wooting-macros/pull/209
* Fixed double element display by @medzernik in https://github.com/WootingKb/wooting-macros/pull/210

**Full Changelog**: https://github.com/WootingKb/wooting-macros/compare/v1.0.2...v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/WootingKb/wooting-macros/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by LeoSeabra** [comment]
I've just installed this on Linux ([Zorin OS 17.1 Pro](https://zorin.com/os/)) and the app justs hangs and runs very slowly, consuming a lot of resources. 

It's a pretty decent machine... 
![image](https://github.com/WootingKb/wooting-macros/assets/65295381/3b3ddae0-dce7-47e0-823b-a67127364531)

**Reply by LeoSeabra** [reply]
Update on this: Installed previous version (1.0.2) and it's rolling fine.
